addons/pos_self_order/static/src/app/pages/cart_page/cart_page.js

```diff
@@ -18,6 +18,7 @@ export class CartPage extends Component {
 
         this.state = useState({
             selectTable: false,
+            sendInProgress: false,
```

Comment:
Why is `sendInProgress` in `state`? Its use in `pay` method, it is accessed from `this`.  Also, `pay` doesn't return early when `sendInProgress` is true.